src/main/java/org/opentripplanner/transit/raptor/package.md

```diff
@@ -154,20 +154,21 @@ stops/routes that can be used as a filter for the _McRR_. _RR_ is super fast, mo
 than McRR with 4 criteria.
 
 ## Understanding the search (range-raptor algorithm implementation)
-The `RangeRaptorWorker` and the `RoutingStrategy` together implement the _range-raptor_ 
-algorithm. There are 3 `RoutingStrategy` implementations:
-1. The `StdTransitWorker` is the standard Range Raptor implementation. Support both _forward_ and 
-_reverse_ search.
-1. The `NoWaitTransitWorker` is the same as the standard, but it eliminates _wait-time_. It support 
-both _forward_ and _reverse_ search. It is very fast, and can be used to compute various 
-heuristics, like _minimum-number-of-transfers_, _minimum-travel-time_ and 
-_earliest-possible-arrival-time_.
-1. The `McTransitWorker` is the _Multi-Criteria Range Raptor_ implementation. It does **not** 
-support _reverse_ search - so far there has not been a need for it.
+The `RangeRaptorWorker` and the `RoutingStrategy` together implement the _range-raptor_ algorithm. 
+There are three `RoutingStrategy` implementations:
+1. The `ArrivalTimeRoutingStrategy` is the standard Range Raptor implementation. It supports both
+   _forward_ and _reverse_ search and is used to find the path with the best _arrival-time_.
+2. The `MinTravelDurationRoutingStrategy` is the same as the standard, but optimize on 
```

Comment:
Again, it could be further explained what eliminating wait-time means in practice.